src/api.ts

```diff
@@ -107,50 +105,48 @@ class RippleAPI extends events.EventEmitter {
       this.connection = new RestrictedConnection(null, options)
     }
   }
-}
 
-_.assign(RippleAPI.prototype, {
-  connect,
-  disconnect,
-  isConnected,
-  getServerInfo,
-  getFee,
-  getLedgerVersion,
+  connect = connect
```

Comment:
Adding these as class properties makes them a part of the interface, vs. `_.assign()` which was appending them to the prototype in a way that confused TypeScript.